src/display/editor/editor.js

```diff
@@ -455,11 +457,13 @@ class AnnotationEditor {
    * @param {number} y - y-translation in page coordinates.
    */
   translateInPage(x, y) {
+    this.#initialPosition ||= [this.x, this.y];
```

Comment:
Don't we need this in the `translate` method, just above, as well?

Reply:
No, because only `translateInPage` and `drag` have been triggered by the user.